athena/tests/module_text_llm/test_basic_approach_mock.py

```diff
@@ -0,0 +1,165 @@
+import sys
+from unittest.mock import Mock, patch
+from pydantic import BaseModel, Field
+from typing import Callable, Any
+import configparser
+import pytest
+from module_text_llm.basic_approach.generate_suggestions import generate_suggestions
+from athena.text import Exercise, Submission, Feedback
+from module_text_llm.approach_config import ApproachConfig
+from athena.schemas.exercise_type import ExerciseType
+from tests.utils.mock_env import mock_sent_tokenize
+from tests.utils.mock_llm import MockLanguageModel, MockAssessmentModel, MockFeedbackModel
+from tests.utils.mock_config import MockApproachConfig, MockModelConfig
+
+# ---- Patch llm_core.models.openai ----
+class MockOpenAIModelConfig(BaseModel):
+    """Mock configuration for OpenAI model settings."""
+    model_name: str = "mock_model"
+    get_model: Callable[[], Any] = Field(default_factory=lambda: Mock())
+
+    class Config:
+        arbitrary_types_allowed = True
+
+mock_openai = Mock()
+mock_openai.OpenAIModelConfig = MockOpenAIModelConfig
+mock_openai.available_models = {'mock_model': Mock()}
+sys.modules['llm_core.models.openai'] = mock_openai
```

Comment:
You already have a dedicated file under utils that does the same thing, `tests/utils/mock_openai.py` 😊